doc/Style.md

```diff
@@ -4,6 +4,45 @@ This document describes the style used within Agora.
 It covers the style itself, which should be clear and unambiguous,
 as well as more design aspects on how we approach / solve problems.
 
+## Commits
+
+- A commit must be a self-contained unit of logical change.
+
+- A commit should not contain unrelated changes mixed into the same commit.
+
+- A commit should be relatively simple to review. If the commit changes
+too many files, or adds too much functionality, consider splitting it
+up into several distinct commits.
+
+- Each commit **must be** compileable and **must** succeed a run of the test-suite.
+
+- The commit title and description should follow some general rules:
+
+An example good commit: https://github.com/bpfkorea/agora/commit/71e7d1cc1da314d92eb76c1d02e04f43bc83784f
```

Comment:
I originally wanted to just link to the article and call it a day, as it does a very good job at explaining what's needed, but I like the focus you put there 👍